src/components/RadioTile/RadioTile.js

```diff
@@ -16,6 +17,7 @@ export default class RadioTile extends React.Component {
 
   static defaultProps = {
     onChange: () => {},
+    className: null,
```

Comment:
Don't need this in `defaultProps` since `classNames` will have similar behavior with `undefined` 😄

Reply:
Thanks for reviewing the PR much appreciated - will make the changes 👍